packages/core/src/modules/proofs/protocol/v2/__tests__/indy-proof-negotiation.test.ts

```diff
@@ -0,0 +1,444 @@
+import type { Agent } from '../../../../../agent/Agent'
```

Comment:
It seems there's thests both in the top level tests and also in the protocol specific directories. What's the difference between them?

Reply:
Yes, I added tests in the `core/tests` first (as it seemed to me more of a e2e test), and later I noticed, that there are test cases in the `__tests__` folder of the protocol, that are covering similar scenarios, so added there as well. But true, they're almost no different

Reply:
Can we remove one of them in that case to keep this tidy? I think my preference would go to keep them in the credentials module folder

Reply:
I assume you mean proofs module folder. Anyway removed those in `core/tests`.

Reply:
Yeah I think this comes from the proofs module, where its root __tests__ directory has some unit tests but protocol/vx includes e2e tests.

I agree on merge these tests and leaving one of each in modules/proofs/__tests__. They can be named 'xxx.e2e.test.ts' like it happens in other modules E2E tests. Later we should open a new PR to harmonize this structure.

Reply:
> I assume you mean proofs module folder

yes 😬